S_02/run.ipynb

```diff
@@ -124,12 +124,15 @@
     "4. Read in the Hamiltonian (as done in [S 1](../S_01/run.ipynb)) and create a Monkhorst-Pack grid as the one in Siesta input. Use this snippet to calculate the PDOS using `sisl`\n",
     "\n",
     "       mp = sisl.MonkhorstPack(H, [nx, ny, 1])\n",
-    "       mp_avg = mp.apply.average\n",
-    "       pdos = mp_avg.PDOS(E, eta=True)\n",
+    "       def wrap_multiple(eigenstate):\n",
+    "           DOS = eigenstate.DOS(E)\n"
+    "           PDOS = eigenstate.PDOS(E)\n"
+    "           return sisl.oplist([DOS, PDOS])\n",
+    "       DOS, PDOS = mp.apply.average.eigenstate(wrap=wrap_multiple, eta=True)\n",
     "\n",
-    "    Search the API documentation for the `MonkhorstPack.apply.average` method and figure out what it does. Note, there are other `MonkhorstPack.apply.*` methods, these are all extremely useful when calculating many quantities of data in a Brillouin-zone object.\n",
+    "    Search the API documentation for the `MonkhorstPack.apply.average` method and figure out what it does. Note, there are other `MonkhorstPack.apply.*` methods - these are all extremely useful when calculating many quantities of data in a Brillouin-zone object.\n",
     "5. Compare the Siesta PDOS with sisl PDOS, why are they different?  \n",
-    "   *HINT*: $\\sigma$. Check the API for the `PDOS` method."
+    "   *HINT*: $\\sigma$. Check the API for the `PDOS` method in the Siesta manual (https://siesta-project.org/SIESTA_MATERIAL/Docs/Manuals/siesta-4.1.5.pdf)."
```

Comment:
I'll have something here, perhaps the shipped siesta manual should be referenced instead of a specific version. Just for clarity.

Reply:
Sure - that makes more sense. However, pointing to the Siesta manual (whichever it is) should be included.